### ISSUE TITLE
Thermal Dynamics crashes server after update.

### ISSUE THREAD

**Issue by crescentrose** [CoFH/Feedback#404]
I've recently updated Thermal Dynamics to the latest version. After the update, I've started getting this crash:

http://pastebin.com/3hbJDqCK

I've went to MCEdit and deleted the entire area around the ducts mentioned in the log, however the game kept crashing on another set of pipes:

http://pastebin.com/QTN7xA1V

What should I do to fix this?

Thanks in advance.

Edit: I should mention that reverting to the older version of Thermal Dynamics did not help. Also I switched the logs - they're in proper order now.

**Comment by KingLemming** [comment]
Literally have no idea. We've never seen this. However, the error is in MFR.

Either way, it's on us.

**Comment by crescentrose** [comment]
Welp. I'll try fiddling with this today. Maybe something got corrupted in the world save.

**Comment by msvisser** [comment]
I can reproduce this crash by updating my FTB Infinity pack to the newest version of Forge. Something seems to have changed between Forge 10.13.2.1291 and 10.13.3.1358.

I haven't yet been able to reproduce it in a world generated in the newest version, but my old world crashes just by updating Forge. Although it's kinda hard to test with it crashing when looking at tanks in the creative inventory, but that is a EnderTanks bug by the looks of it.

Here's one of many crashlogs that say exactly the same:
http://pastebin.com/gpaHzibU

**Comment by crescentrose** [comment]
Okay, I can confirm now that this seems to be related to the latest version of Forge.

I've reverted to a backup, with the same crash happening. I then deleted the entire region which was triggering the crash which regenerated it, and got an entirely new error message completely unrelated to any mods:

http://pastebin.com/P5xMCExX

I'll cross-post this to the Forge repository. Sorry to waste your time, KingLemming.
